Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Pandaria Post-Release - Wandering Isle / Ankoan

## Changelog:
- The Ankoan can now appear in the courts of non-Ankoan followers of religions such as Waterspeaker, Water Deities and Neptulon.
- Around ingame year 610, the Wandering Isle's existence once again becomes known to the public. This event opens a possibility for the Islander Pandaren to appear in various courts in Azeroth, provided you're not a follower of an evil religion. This event also precedes a bigger discovery that happens shortly after...
- Added Sky Burial for Yaungol that can be used at any time regardless of religion if playing that race.
- Reshuffled IDs for Pandaria events, the new namespace is `WCPND`

## Developer changelog:
- Event for Ankoan courtiers, each with his own particular flavor (diplomat, administrator, spy, shaman, hero). Their availablity is regulated by the appearance of the Naga, and they can appear at anytime in 603-605 startdates, since the global flag is active by default in these two startdates.
- Event for Pandaren courtiers, each with his own particular flavor (poet, mason, priest, brewmaster). Their availablity is regulated by the appearance of Shen-zin Su/Wandering Isle. If MoP gamerule is disabled, the requirement is removed, as Shen-zin Su event can only appear if the Mists of Pandaria gamerule is enabled.

## TODO:
- [x] Wandering Isle Sighted event (enables Pandaren courtiers)
   - [x] Add event that can spawn Wandering Isle (Shen-zin) pandaren courtiers in the world
- [x] Add event that can spawn Ankoan courtiers for Jinyu or Nerglish characters after Terror of the Tides
- [x] Ankoan portraits

## How to test:
- Verify if the event's localization appears correctly for Pandarians and non-pandarians
- Verify if the courtiers appear correctly (pandas after the isle event, ankoan after naga appear)
